redfish-core/include/error_messages.hpp

```diff
@@ -482,6 +482,41 @@ nlohmann::json propertyValueIncorrect(std::string_view arg1,
 void propertyValueIncorrect(crow::Response& res, std::string_view arg1,
                             std::string_view arg2);
 
+/**
+ * @brief Formats PropertyValueResourceConflict message into JSON
+ * Message body: "The property '%1' with the requested value of '%2' could
+ * not be written because the value conflicts with the state or configuration
+ * of the resource at '%3'."
+ *
+ * @param[in] arg1 Parameter of message that will replace %1 in its body.
+ * @param[in] arg2 Parameter of message that will replace %2 in its body.
+ * @param[in] arg3 Parameter of message that will replace %3 in its body.
+ *
+ * @returns Message PropertyValueResourceConflict to JSON */
+nlohmann::json propertyValueResourceConflict(const std::string& arg1,
```

Comment:
Why can't this go upstream?

Reply:
And the rest of these in this file..

Reply:
When we discussed early on this, I heard that we cannot push the registry without any use in the upstream bmcweb that is, we are only using it for the hw-isolation so its stays in the downstream

Reply:
> When we discussed early on this, I heard that we cannot push the registry without any use in the upstream bmcweb that is, we are only using it for the hw-isolation so its stays in the downstream


Why, can't we push to the registry ? I don't see a problem with that, hopefully one day this will be upstream

Reply:
Even I am not seeing any problem pushing registry-specific changes upstream (remembered like some process in the past so if something changed I am happy with it :)) - IMO, we can try that in parallel if no restriction now.